app/src/Entity/ExpenseItem.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Entity;
+
+use ApiPlatform\Metadata\ApiResource;
+use App\Repository\ExpenseItemRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: ExpenseItemRepository::class)]
+#[ApiResource]
+class ExpenseItem
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $title = null;
+
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    private ?string $description = null;
+
+    #[ORM\Column(type: Types::DECIMAL, precision: 10, scale: '0')]
+    private ?string $price = null;
+
+    #[ORM\Column]
+    private ?\DateTimeImmutable $created_at = null;
```

Comment:
Remove backslash here -> import DateTimeImmutable via use statement.

Use camelCase with variable names. Change this to $createdAt.

createdAt cannot be nullable. Your object must always have the creation date.